tests/integration/test_migration_workflow.py

```
@@ -0,0 +1,429 @@
+        """Test migration with security validation enabled."""
+        # Create malicious migration request
+        malicious_request = {
+            "compose_dataset": "authelia",
+            "target_host": "host; rm -rf /",  # Command injection attempt
+            "target_base_path": "/home/user/docker",
+            "ssh_user": "root",
+            "ssh_port": 22
+        }
+        
+        # This should fail during validation
+        with pytest.raises(Exception):  # Security validation should raise exception
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)